charts/accurate/templates/certificate.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ template "accurate.fullname" . }}-serving-cert
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "accurate.labels" . | nindent 4 }}
+spec:
+  dnsNames:
+    - {{ template "accurate.fullname" . }}-webhook-service.{{ .Release.Namespace }}.svc
+    - {{ template "accurate.fullname" . }}-webhook-service.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
Don't you have to define the cluster domain (e.g. `cluster.local`) as a variable?

Reply:
I guess this is from the kubebuilder's scaffold.